packages/codegen/generate.hooks.ts

```diff
@@ -242,6 +244,33 @@ export function generateReactHooks<
         }
     }
 
+    function useAllObjects (variables: QueryVariables, options?: QueryHookOptions<IUseObjectsQueryReturnType<GQLObject>, QueryVariables>) {
```

Comment:
Strange hook. What's it's purpose? I think we should not provide this "network-eater" tool globally